<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Test</class>
 <widget class="QMainWindow" name="Test">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>610</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>790</width>
    <height>610</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>790</width>
    <height>610</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: &quot;#455A64&quot;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>341</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>341</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: &quot;#78909C&quot;;
border: 3px solid gray;
border-radius: 25px</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="import_img">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    color: #333;
    border: 2px solid #555;
    border-radius: 10;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #888
        );
    padding: 5px;
    }</string>
         </property>
         <property name="text">
          <string>Import Image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="import_folder">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    color: #333;
    border: 2px solid #555;
    border-radius: 10;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #888
        );
    padding: 5px;
    }</string>
         </property>
         <property name="text">
          <string>Import Folder</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="run_test">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    color: #333;
    border: 2px solid #555;
    border-radius: 10;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #888
        );
    padding: 5px;
    }</string>
         </property>
         <property name="text">
          <string>GO</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="back_button">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    color: #333;
    border: 2px solid #555;
    border-radius: 10;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #888
        );
    padding: 5px;
    }</string>
         </property>
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>import_img</zorder>
      <zorder>run_test</zorder>
      <zorder>import_folder</zorder>
      <zorder>back_button</zorder>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QTextBrowser" name="consol">
      <property name="minimumSize">
       <size>
        <width>760</width>
        <height>192</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>691</width>
        <height>192</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: &quot;transparent&quot;;
border: 3px solid gray;
border-radius: 25px;
color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">background-color: &quot;transparent&quot;;
border: 3px solid gray;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>790</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>import_folder</tabstop>
  <tabstop>import_img</tabstop>
  <tabstop>consol</tabstop>
  <tabstop>run_test</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
